### PR TITLE
Added Adultery, Desertion and exit scenarios

### PR DESCRIPTION
**Pull request by marianadpereira**
# Description

Added missing paths to Adultery and Desertion for E2E english journey.
Also added already existing exit paths for 5 years separation.

# How Has This Been Tested?

Tested locally and on AAT.

### CODE REVIEW

**Review by mhisltd — APPROVED** [review]
LGTM - Approved

File: test/end-to-end/pages/prayer/claim-costs.js [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
-
   I.waitInUrl(pagePath, 5);
   I.seeCurrentUrlEquals(pagePath);
   I.checkOption(content.yes);
   I.navByClick('Continue');
 }
-
-module.exports = { enterClaimCosts };
\ No newline at end of file
+function enterClaimCostsCorrespondent() {
```
Comment (mounikahmcts) [inline test/end-to-end/pages/prayer/claim-costs.js:10]: A new line between functions for readability

File: test/end-to-end/pages/prayer/claim-costs.js [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
+  const I = this;
+  I.waitInUrl(pagePath, 5);
+  I.seeCurrentUrlEquals(pagePath);
+  I.checkOption(content.yes);
+  I.checkOption( '#correspondent');
+  I.navByClick('Continue');
+}
+module.exports = { enterClaimCosts, enterClaimCostsCorrespondent };
```
Comment (mounikahmcts) [inline test/end-to-end/pages/prayer/claim-costs.js:18]: A new line between functions for readability

File: test/end-to-end/paths/adulteryAndDesertionJourneys.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+  I.selectAdulteryWhen();
+  I.enterAdulteryDetails();
+  I.enterAdulterySecondHandInfo();
+
+  I.enterLegalProceedings();
+  I.selectFinancialArrangements();
+  I.enterFinancialAdvice();
+  I.enterClaimCostsCorrespondent();
+
+  if(['safari', 'microsoftEdge'].includes(config.features.browserSupport)) {
```
Comment (mounikahmcts) [inline test/end-to-end/paths/adulteryAndDesertionJourneys.js:54]: https://standardjs.com/rules.html - No spacing between if and (

File: test/end-to-end/paths/adulteryAndDesertionJourneys.js [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+  const isDragAndDropSupported = await I.checkElementExist('.dz-hidden-input');
+  I.uploadMarriageCertificateFile(isDragAndDropSupported);
+
+  await I.completeEquality();
+
+  I.checkDesertionDateOnCYAPage();
+  I.checkMyAnswers();
+
+  const genericErrorPage = await I.checkElementExist('//h1[contains(text(), \'There has been a problem\')]');
+  if(genericErrorPage) {
```
Comment (mounikahmcts) [inline test/end-to-end/paths/adulteryAndDesertionJourneys.js:119]: same here and for else as well

**Review by mounikahmcts — APPROVED** [review]
Functionally tests are ok to me. Please follow standards as per https://standardjs.com/rules.html

**Review by ruban72 — APPROVED** [review]

**Pull request merged**